File: leo2-common/src/main/java/org/deku/leo2/rest/entities/v1/DayType.java
package org.deku.leo2.rest.entities.v1;

/**
 * Created by JT on 02.06.15.
 */
public enum DayType {
//    unknown(0),
    WorkDay("WorkDay"),
    Saturday("Saturday"),
    Sunday("Sunday"),
    Holiday("Holiday"),
    RegionalHoliday("RegionalHoliday");

    String mType;

    DayType(String type) {
        mType = type;
    }
}

File: leoz-common/src/main/java/org/deku/leoz/service/entity/pub/DayType.kt
package org.deku.leoz.service.entity.pub

/**
 * Created by JT on 02.06.15.
 */
enum class DayType private constructor(private var type: String) {
    Workday("Workday"),
    Saturday("Saturday"),
    Sunday("Sunday"),
    Holiday("Holiday"),
    RegionalHoliday("RegionalHoliday")
}